### PR TITLE
Feat/Support `neofs-adm`

### LINKED ISSUE #197: Provide `dev-env` configuration that could be supported by `neofs-adm`
`neofs-adm` could be used in `dev-env` (without init steps and so on) but currently every developer should change (rename, move, create additional dirs) repo to be able to use it for testing.

Provide config that satisfies `neofs-adm` requirements and document it (describe what it is for, how to use, etc).

### PR DESCRIPTION
**Pull request by carpawell**
Closes #197.

1. Rename the alphabet wallet to satisfy `neofs-adm`
2. Add `contract.json` to have `neofs.group` (well, yes, it turned out that group key and the Alphabet key are the same in `dev-env`) public
3. Remove strange acc with 3/4 multisig from Alphabet's wallet (i do not know that it is and how that could be used in that repo), added labels to the existing accs.
4. Added short desc (do not know who is going to read that and what info he is going to get from reading, suggestions are appreciated)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,12 @@ You can find more information on each service in `docs` directory.
 
 Maybe you will find the answer for your question in [F.A.Q.](docs/faq.md)
 
+## Using NeoFS Admin Tool in `dev-env`
+
+Devenv support administration NeoFS network via [neofs-adm](https://github.com/nspcc-dev/neofs-node/tree/master/cmd/neofs-adm).
```
Comment (fyrchik) [inline README.md:126]: `NeoFS network management`?

File: README.md [hunk 2]
```diff
@@ -121,6 +121,12 @@ You can find more information on each service in `docs` directory.
...
+`services/ir` contains the Alphabet wallet that corresponds to the utility format, specify it
```
Comment (fyrchik) [inline README.md:127]: `in a proper format`?

**Review by fyrchik — APPROVED** [review]

**Comment by alexvanin** [comment]
Don't you need to include neofs-adm config file with pre-configured path's and passwords?

**Comment by carpawell** [comment]
@alexvanin, good idea, added config file with endpoint and creds for wallets.

**Review by fyrchik — APPROVED** [review]

**Pull request merged**